### PR TITLE
Hidden columns

### PR DESCRIPTION
**Pull request by PolGuixe**
### Description

Added classes to hide columns:

- `col-xs-hidden`
- `col-sm-hidden`
- `col-md-hidden`
- `col-lg-hidden`

### Check List
__instruction : terminal command__
- [x] run the build script `grunt`
- [x] open  `index.html` in a browser & resize to test visual issues

### CODE REVIEW

**Comment by lgg** [comment]
Maybe change classnames to fit bootstrap API?

**Comment by PolGuixe** [comment]
@lgg good point. Done.

**Comment by msiemens** [comment]
Any news on this?

**Comment by PolGuixe** [comment]
@msiemens I think the branch is good to be merged. Although, there are some problems with Travis. @kristoferjoseph according to Travis it is unable to find the config file.

**Comment by Falieson** [comment]
Citing this PR for my inclusion of cols={0} into react-flexbox-grid https://github.com/roylee0704/react-flexbox-grid/pull/82

**Comment by menosprezzi** [comment]
Acording to this link https://travis-ci.org/kristoferjoseph/flexboxgrid/pull_requests, seems like the travis build system is not setup well @kristoferjoseph. Maybe disable the travis build for this tasks is better

**Comment by bgarrant** [comment]
Has this been merged into latest branch? I really need a few of these visibility classes.

**Comment by lgg** [comment]
@bgarrant until this PR isn't merged you can use: https://github.com/lgg/flexboxgrid-helpers

**Comment by PolGuixe** [comment]
@kristoferjoseph what do we need to do to get this approved and merged?

**Comment by PolGuixe** [comment]
@bgarrant [this](https://github.com/Zetoff/flexboxgrid/tree/hidden-columns) is the original branch with the hidden classes if you want to use it in the mean time.

**Comment by bgarrant** [comment]
Thanks PolGuixe and lgg!

**Comment by Falieson** [comment]
so what needs to be done to get this PR approved?

**Comment by b264** [comment]
We should all decide on a fork to concentrate on if this can't be merged...

**Comment by lgg** [comment]
ping @kristoferjoseph

**Comment by hassanasad** [comment]
I think its a dead project now :( - None of the PR's are getting merges nor are any new commits coming in. Can someone please fork and maintain this project?

**Comment by b264** [comment]
Do it! and reference this PR so others know about it :sunglasses:

**Comment by Falieson** [comment]
@PolGuixe could you re-submit the PR to my fork ? https://github.com/Falieson/flexboxgrid

@b264 @hassanasad @lgg @kristoferjoseph  I'll step up as maintainer, and @thehappyginger will help me

**Comment by PolGuixe** [comment]
I'll bé quite busy this week I'll do my best :-).

Pol Guixe

**Comment by Falieson** [comment]
@PolGuixe @b264 @hassanasad 

I was able to get ahold of @kristoferjoseph and he has said 
1) this repo is still being actively maintained 
2) all these addon features we want he believes are handled by @lgg 's https://github.com/lgg/flexboxgrid-helpers
3) these features shouldn't be incorporated into flexboxgrid b/c he wants to keep the package size down

Speak up if you have an issue with this resolution, I may rename the fork something like 'flexboxgrid-extended' if there is good reason to have a combined package but I think everyone should just be able to depend on the 2 packages.

**Comment by b264** [comment]
If https://github.com/kristoferjoseph/flexboxgrid/pull/246 merges, that sounds great

**Comment by hassanasad** [comment]
@Falieson, @kristoferjoseph seems to have a valid point - However, some of the pull requests (such as XL breakpoint support and few helpers such as this) might not add that much weight to the package and they are kinda like becoming a standard requirement for a grid system.

To be honest, in my personal opinion - when the css grid layout rolls out in main stream browsers some of the code from these packages can be cleared out to bring it back to a very small sized package.

**Comment by Falieson** [comment]
@hassanasad I agree that I expect some features to be apart of the standard offering, but that's because my first grid was http://materializecss.com/grid.html

**Comment by b264** [comment]
Without at a minimum https://github.com/kristoferjoseph/flexboxgrid/pull/246 merging, it does not appear this repo is maintained.

**Comment by cyberhck** [comment]
Why is this PR not merged? isn't this supposed to be a standard feature? Is this going to get merged or not? If it's not gonna get merged, simply closing the PR would be better, if it's gonna get merged, some activity would have been nice.

**Comment by b264** [comment]
@lgg TL;DR: This repo is unmaintained and abandoned

**Comment by cyberhck** [comment]
@b264 you seem to be very correct :)

**Comment by lgg** [comment]
@cyberhck @b264 no, you are wrong. This repo is maintained, but @kristoferjoseph don't want to add  current feature to code.

**Comment by cyberhck** [comment]
@lgg , I was looking for react-flexbox-grid, because of not adding basic feature to this repo, I've to find a entirely new grid library written for react, has visibility classes AND has at least typescript definition.

**Comment by b264** [comment]
@lgg Here is part of the !TL;DR version:  It's not maintained if a simple documentation clarification https://github.com/kristoferjoseph/flexboxgrid/pull/246 can't be merged.  There are no reasons to not merge or _at least review_ helpful documentation which has been submitted as a pull request, aside from the repo not being maintained.  It really is that simple.  Do we want to keep going?  Clearly, you have a goal in helping @kristoferjoseph making it look like this is maintained but alas it is not.  Why do you fool yourself?
  Everything here is write-only. https://github.com/kristoferjoseph/flexboxgrid/issues/248 @cyberhck  If @kristoferjoseph really feels how he wrote he does, then he, at a minimum, could update documentation, or allow others to update it, indicating that the source code won't ever be changed.  But, he doesn't even do that.  He does nothing.  If you feel that "unmaintained" has some other definition that everyone else here is unaware of, please do clarify exactly what it means.  Because my dew definition of "unmaintained" is "like kristoferjoseph/flexboxgrid"

TL;DR: This repository is a poster child for a repo being unmaintained.

**Comment by hassanasad** [comment]
Please everyone move to https://github.com/evgenyrodionov/flexboxgrid2 its fresh and being maintained.

**Comment by evgenyrodionov** [comment]
Hidden utils classes implemented in [flexboxgrid2#7.0.0-alpha7](https://github.com/evgenyrodionov/flexboxgrid2/releases/tag/7.0.0-alpha7) 😌

**Comment by cyberhck** [comment]
@b264 You're absolutely correct, this repo is not maintained. There's not a single comment on #246 or this PR from owner of this repo. Who's maintaining it then? And last commit was almost 8 months ago as of this writing. That's the definition of not being maintained.
